sktime/transformations/series/tests/test_boxcox.py

```diff
@@ -25,9 +26,38 @@ def test_boxcox_against_scipy():
 
 
 @pytest.mark.parametrize("bounds", [(0, 1), (-1, 0), (-1, 2)])
-@pytest.mark.parametrize("method", ["mle", "pearsonr"])
-def test_lambda_bounds(bounds, method):
+@pytest.mark.parametrize(
+    "method, sp", [("mle", None), ("pearsonr", None), ("guerrero", 5)]
+)
+def test_lambda_bounds(bounds, method, sp):
     y = load_airline()
-    t = BoxCoxTransformer(bounds=bounds, method=method)
+    t = BoxCoxTransformer(bounds=bounds, method=method, sp=sp)
     t.fit(y)
     assert bounds[0] < t.lambda_ < bounds[1]
+
+
+@pytest.mark.parametrize(
+    "bounds, r_lambda",
+    [
+        ((0, 1), 6.61069613518961e-05),
+        ((-1, 0), -0.156975034727656),
+        ((-1, 2), -0.156981228426408),
+    ],
+)
+def test_guerrero_against_r_implementation(bounds, r_lambda):
+    """
+    Testing lambda values estimated by the R implementation of the Guerrero method
+    https://github.com/robjhyndman/forecast/blob/master/R/guerrero.R
+    against the guerrero method in BoxCoxTransformer.
+    R code to generate the hardcoded value for bounds=(-1, 2) used in the test
+    ('Airline.csv' contains the data from 'load_airline()'):
+        airline_file <- read.csv(file = 'Airline.csv')[,c('Passengers')]
+        airline.ts <- ts(airline_file)
+        guerrero(airline.ts, lower=-1, upper=2, nonseasonal.length = 20)
+    Output:
+        -0.156981228426408
+    """
+    y = load_airline()
+    t = BoxCoxTransformer(bounds=bounds, method="guerrero", sp=20)
+    t.fit(y)
+    assert t.lambda_ == approx(r_lambda, abs=1e-4)
```

Comment:
Oh nice didn't know that pytest provides `approx` but `np.testing` seems to be preferred in the data science ecosystem, e.g. `np.testing.assert_almost_equal`

Reply:
Ok thanks, I'll replace `approx` with `np.testing` :)

Reply:
Hi @mloning!
I've replaced `approx` with `np.testing.assert_almost_equal`.